packages/frontend/package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "preview": "vite preview",
-    "build": "vite build",
+    "build": "(cp ../../README.md src || copy ../../README.md src) && vite build",
```

Comment:
do we need to copy the README file ? we can't import the file from the parent folder ?
Also if you need to copy you should probably use an npmjs package for that rather than one for windows and one for mac/linux